source/custom/nifake_non_ivi_service.custom.cpp

```diff
@@ -6,7 +6,7 @@
 #include <thread>
 namespace nifake_non_ivi_grpc {
 
-::grpc::experimental::ServerWriteReactor<ReadStreamResponse>* NiFakeNonIviService::ReadStream(::grpc::CallbackServerContext* context, const ReadStreamRequest* request)
+::grpc::ServerWriteReactor<ReadStreamResponse>* NiFakeNonIviService::ReadStream(::grpc::CallbackServerContext* context, const ReadStreamRequest* request)
 {
   // Wraps a thread to call join on destruct.
   struct AutoJoinThread {
```

Comment:
Note: It's hard to see the actual `grpc` update below.

The easiest way to see it is [the files view for my branch](https://github.com/astarche/grpc-device/tree/users/astarche/grpc-139/third_party).

Steps (approximate):
```
cd third_party/grpc
checkout v1.39
cd ../..
<git add/commit/push>
```